packages/spatial/src/input/functions/ClientInputHooks.tsx

```diff
@@ -348,7 +351,7 @@ export const CanvasInputReactor = () => {
       canvas.removeEventListener('click', onClick)
       canvas.removeEventListener('wheel', onWheelEvent)
     }
-  }, [xrState.session])
+  }, [xrState.session, rendererComponent])
```

Comment:
we probably want this to be rendererComponent.canvas, rather than potentially reacting to anything changing in the renderer component

Reply:
Good point